UnitySDK/Assets/ML-Agents/Scripts/LearningBrain.cs

```diff
@@ -50,16 +49,6 @@ public class LearningBrain : Brain
         private IReadOnlyList<TensorProxy> m_InferenceInputs;
         private IReadOnlyList<TensorProxy> m_InferenceOutputs;
 
-        /// <summary>
-        /// When Called, the brain will be controlled externally. It will not use the
-        /// model to decide on actions.
-        /// </summary>
-        public void SetBatcher(Batcher batcher)
```

Comment:
In my previous PR, I removed the capacity of Non-Learning Brains to have a communicator. It seems you are re-introducing it here by giving the abstract Brain the m_Communicator. Please make this a LearningBrain thing only.

Reply:
done.